src/main/java/com/synopsys/integration/alert/provider/blackduck/tasks/BlackDuckAccumulator.java

```diff
@@ -77,6 +80,7 @@ public String formatDate(OffsetDateTime date) {
 
     @Override
     public void runProviderTask() {
+        blackDuckValidator.validate(getProviderProperties());
```

Comment:
I would check the boolean value returned by validate.  If validate returns true then I would call the accumulate method.  Otherwise don't bother trying to accumulate.

Reply:
This is just an enhancement not a blocking issue for this PR.